TFT/src/User/Menu/BedLeveling.c

```diff
@@ -82,7 +96,12 @@ void menuBedLeveling(void)
         break;
 
       case KEY_ICON_1:
-        infoMenu.menu[++infoMenu.cur] = menuMeshEditor;
+        #if DELTA_PROBE_TYPE == 2  // if Delta printer with removable probe
```

Comment:
invert the logic. The default matching condition should be for not removable probe. so #if DELTA_PROBE_TYPE != 2 then use old logic #else use the new logic,